scripts/tests/test_run_event_benchmark_matrix.py

```
@@ -0,0 +1,1340 @@
+        with tempfile.TemporaryDirectory() as tmp:
+            calls = []
+            result = harness.run_paired_trial_plan(
+                "model",
+                plan,
+                side_a,
+                side_b,
+                log_dir=Path(tmp),
+                trial_executor=self._make_fake_executor(harness, calls),
+            )
+        self.assertEqual(len(result.executed_order), len(plan))
+        for order_entry, plan_entry in zip(result.executed_order, plan):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)